token/cli/src/config.rs

```diff
@@ -307,14 +307,18 @@ impl<'a> Config<'a> {
         override_name: &str,
         wallet_manager: &mut Option<Arc<RemoteWalletManager>>,
     ) -> Result<Pubkey, Error> {
-        let token = pubkey_of_signer(arg_matches, "token", wallet_manager).unwrap();
-        self.associated_token_address_for_token_or_override(
-            arg_matches,
-            override_name,
-            wallet_manager,
-            token,
-        )
-        .await
+        match pubkey_of_signer(arg_matches, "token", wallet_manager) {
```

Comment:
nit: we can keep this roughly the same by mapping the error, ie
```suggestion
        let token = pubkey_of_signer(arg_matches, "token", wallet_manager).map_err(|e| e.to_string().into())?;
```